packages/web/src/app/[domain]/search/page.tsx

```
@@ -95,12 +98,11 @@ const SearchPageInternal = () => {
         queryKey: ["repos"],
         queryFn: () => getRepos(domain),
         select: (data): Record<string, Repository> =>
-            data.List.Repos
-                .map(r => r.Repository)
+            data.repos
                 .reduce(
                     (acc, repo) => ({
                         ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)